<launch>
  <include file="$(find-pkg-share apex_putter)/launch/vision.launch.xml">
  </include>

  <include file="$(find-pkg-share easy_handeye2)/launch/calibrate.launch.py">
    <arg name="name" value="calib" />
    <!-- <arg name="freehand_robot_movement" value="true" /> -->
    <arg name="calibration_type" value="eye_on_base" />

    <arg name="robot_base_frame" value="/base" />
    <arg name="robot_effector_frame" value="/fer_link8" />

    <arg name="tracking_base_frame" value="/camera_color_optical_frame"/>
    <arg name="tracking_marker_frame" value="/tag_36" />
  </include>
</launch>
